game/songs.cc

```diff
@@ -61,7 +61,7 @@ void Songs::reload_internal() {
 	m_loading = false;
 	std::clog << "songs/notice: Done Loading. Loaded " << m_songs.size() << " Songs." << std::endl;
 	Game* gm = Game::getSingletonPtr();
-	gm->dialog(_("Done Loading! Loaded ") + boost::lexical_cast<std::string>(m_songs.size()) + " Songs.");	
+	gm->dialog(_("Done Loading!\n Loaded ") + boost::lexical_cast<std::string>(m_songs.size()) + " Songs.");	
```

Comment:
I wouldn't post code straight to master. Even though it's as small as changing a string.. First check than merge :P